src/main/java/com/greenfox/poker/model/Action.java

```diff
@@ -0,0 +1,21 @@
+package com.greenfox.poker.model;
+
+
+public enum Action {
+  RAISE("raise"), CHECK("check"), CALL("call"), FOLD("fold"), STANDUP("standup");
+
+  private final String actionName;
+
+  Action(String s){
+    this.actionName = s;
+  }
+
+  public String getActionName() {
+    return actionName;
+  }
+
+  @Override
+  public String toString() {
+    return super.toString();
```

Comment:
If you don't modify the built in toString method, you shouldn't override it by its default functionality...